editor/src/components/editor/store/permissions.ts

```diff
@@ -0,0 +1,14 @@
+import { useIsLoggedIn } from '../../../core/shared/multiplayer-hooks'
+import { useIsMyProject } from './collaborative-editing'
+
+export type Permissions = {
+  edit: boolean // Edit the open project via the canvas, inspector, code editor
+  comment: boolean // Add comments
+}
+
+export function usePermissions(): Permissions {
+  return {
+    edit: useIsMyProject(),
+    comment: useIsLoggedIn(),
```

Comment:
👍 
After this PR is merged, I'm going to replace the isLoggedIn guards for the comment ui to use this permission flag.